crates/bitwarden-api-key-connector/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "bitwarden-api-key-connector"
```

Comment:
❓ Which team is going to be owning the new crate? I assume KM. Please add this to the CODEOWNERS